geomesa-fs/geomesa-fs-spark/src/main/scala/org/locationtech/geomesa/fs/spark/FileSystemRDDProvider.scala

```diff
@@ -48,45 +49,74 @@ class FileSystemRDDProvider extends SpatialRDDProvider with LazyLogging {
     val ds = DataStoreFinder.getDataStore(params.asJava).asInstanceOf[FileSystemDataStore]
     try {
       val sft = ds.getSchema(query.getTypeName)
-
       val storage = ds.storage(query.getTypeName)
 
-      // split up the job by the filters required for each partition group
-      val filters = storage.getPartitionFilters(query.getFilter).flatMap { fp =>
-        val paths = fp.partitions.flatMap(storage.getFilePaths)
-        if (paths.isEmpty) { Seq.empty } else {
-          Seq(fp.filter -> paths)
-        }
-      }
-
-      def runQuery(filter: Filter, paths: Seq[Path]): RDD[SimpleFeature] = {
+      def runQuery(filter: Filter, paths: Seq[StorageFilePath], modifications: Boolean): RDD[SimpleFeature] = {
         // note: file input format requires a job object, but conf gets copied in job object creation,
         // so we have to copy the file paths back out
         val job = Job.getInstance(conf)
 
-        // Note we have to copy all the conf twice?
-        FileInputFormat.setInputPaths(job, paths: _*)
+        // note: we have to copy all the conf twice?
```

Comment:
Why's this a question?:)

Reply:
not sure, think that was a comment from @jahhulbert-ccri